from mz_platform.exceptions import mz_exception


class MZServiceException (mz_exception.MZException):
    pass
